unit_tests/engine_test_helper.h

```diff
@@ -13,11 +13,16 @@
 #include "main_trigger_callback.h"
 #include "unit_test_framework.h"
 
+class EngineTestHelperBase
+{
+public:
+	EngineTestHelperBase();
```

Comment:
what is this and how is this helping to fix the test? Was it supposed to be part of the PR?

Reply:
Yes, the idea is to set timeNowUs=0 before calling the EngineTestHelper ctor (and, more important, the Engine ctor).